src/objects/aff/zcl_abapgit_object_ront.clas.xml

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<abapGit version="v1.0.0" serializer="LCL_OBJECT_CLAS" serializer_version="v1.0.0">
+ <asx:abap xmlns:asx="http://www.sap.com/abapxml" version="1.0">
+  <asx:values>
+   <VSEOCLASS>
+    <CLSNAME>ZCL_ABAPGIT_OBJECT_RONT</CLSNAME>
+    <LANGU>E</LANGU>
+    <DESCRIPT>ABAPGIT Objecthandler for EVBT</DESCRIPT>
```

Comment:
The class description should be `RONT - SAP Object Type` (see other `zcl_abapgit_object_*` classes)